src/main/java/uk/gov/hmcts/ccd/domain/service/accessprofile/filter/RoleAssignmentsFilteringServiceImpl.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.ccd.domain.service.accessprofile.filter;
```

Comment:
```suggestion
package uk.gov.hmcts.ccd.domain.service.casedataaccesscontrol;
```

Reply:
same for the matchers. 
I think we can have all case data access control service classes under:
`uk.gov.hmcts.ccd.domain.service.casedataaccesscontrol`